### ISSUE TITLE
CI

### ISSUE THREAD

**Issue by alexgleason** [nostr-dev-kit/ndk#160]
Should a GitHub actions file be added to this project?

**Comment by erskingardner** [comment]
Yes. I'm working towards getting things a bit more in order right now (updating deps, fixing and adding tests, etc). As soon as I'm a bit further along, I'll add some actions and CI.
